### PR TITLE
Unit updates for consistency w/ SIMA

### PR DESCRIPTION
**Pull request by peverwhee**
In bringing in new framework updates, the unit conversion abilities are coming along as well! The result is that unit differences actually matter now. Here are some unit fixes for atmospheric_physics.

PR also brings in standard name updates to align with the dictionary

### CODE REVIEW

**Review by nusbaume — APPROVED** [review]
Looks great, thanks @peverwhee!

File: utilities/qneg.meta [hunk 1]
```diff
@@ -133,13 +133,13 @@
   type  = scheme
 [ mpi_communicator ]
   standard_name = mpi_communicator
-  units = 1
+  units = index
   type = integer
   dimensions = ()
   intent = in
 [ rootprocid ]
```
Comment (cacraigucar) [inline utilities/qneg.meta:140]: Isn't this mpi_root (based on what is in the CCPP standard dictionary)?

File: utilities/qneg.meta [hunk 2]
```diff
@@ -61,7 +61,7 @@
   intent = in
 [ q ]
   standard_name = ccpp_constituents
-  units = kg kg-1
+  units = none
```
Comment (cacraigucar) [inline utilities/qneg.meta:64]: I was told not that long ago (I think) that I should be using  kg kg-1 for ccpp_constituents - that's what I'm doing for the zm code.

**Review by cacraigucar — CHANGES_REQUESTED** [review]
Both of my comments apply to a couple of locations each if changes need to be made

File: utilities/qneg.meta (hunk 2 above)
Comment (peverwhee) [reply utilities/qneg.meta:64]: I think it needs to be none. That's what's in the dictionary and the units are (at least currently) determined by the constituent properties array

File: utilities/qneg.meta (hunk 1 above)
Comment (peverwhee) [reply utilities/qneg.meta:140]: I think you're looking at the local name - the standard name is indeed mpi_root

Comment (cacraigucar) [reply utilities/qneg.meta:140]: Yup - you are correct

File: utilities/qneg.meta (hunk 2 above)
Comment (cacraigucar) [reply utilities/qneg.meta:64]: OK - I'll change my ZM metadata files as well.

**Review by cacraigucar — APPROVED** [review]

**Pull request merged**